assets/locales/en/translation.json

```diff
@@ -58,7 +58,12 @@
       "Per Row": "Per Row",
       "Column: {{column}}": "Column: {{column}}",
       "Columns: {{columns}}": "Columns: {{columns}}",
-      "Click here to analyze in Metabase": "Click here to analyze in Metabase"
+      "TableDropdown": {
+        "New Question": "New Question",
+        "New SQL query": "New SQL query",
```

Comment:
"New Question" is too vague and misleading. We want an "Overview" or "Examples" page.

Reply:
as discussed earlier - this would be too easy to break the query in the GUI, so removing